projects/golang/go/1.17/rpmbuild/SPECS/golang.spec

```diff
@@ -157,6 +157,7 @@ Patch1:		0001-go-1.17.13-eks-archive-tar-limit-size-of-head.patch
 Patch2:		0002-go-1.17.13-eks-net-http-httputil-avoid-query-parameter-smuggling.patch
 Patch3:		0003-go-1.17.13-eks-regexp-limit-size-of-parsed-regexps.patch
 Patch4:     0004-go-1.17.13-eks-syscall-os-exec-reject-environ.patch
+Patch5:     0005-go1.17.13-eks-os-net-http-avoid-escapes-from-os.DirF.patch
```

Comment:
```suggestion
Patch5:     0005-go1.17.13-eks-os-net-http-avoid-escapes-from-os.patch
```

Reply:
Not sure if this is a problem? Haven't seen this format before. The presubs pass here, but failed in 1.16 which has the same thing so maybe this isn't concerning.

Reply:
I don't think it is. But I can try renaming the file and see if that fixes it